src/VolleyManagement.Backend/VolleyManagement.UI/GlimpseSecurityPolicy.cs

```diff
@@ -1,4 +1,5 @@
 // Uncomment this class to provide custom runtime policy for Glimpse
+#pragma warning disable S125 // Sections of code should not be "commented out"
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)